### PR TITLE
Libssh test fix

### PR DESCRIPTION
**Pull request by Jakuje**
#### Description

Minor fixes to the libssh integration test

#### Reviewer's checklist:

- [x] Any issues marked for closing are addressed
- [x] There is a test suite reasonably covering new functionality or modifications
- [ ] This feature/change has adequate documentation added
- [x] Code conform to coding style that today cannot yet be enforced via the check style test
- [x] Commits have short titles and sensible commit messages
- [ ] Coverity Scan has run if needed (code PR) and no new defects were found

### CODE REVIEW

**Comment by Jakuje** [comment]
sigh ... sounds like the engine.h is included in two places but we removed only one. Will fix with another MR on libssh side.

**Comment by Jakuje** [comment]
Pushed in here with some more assorted stuff:
https://gitlab.com/libssh/libssh-mirror/-/merge_requests/509

File: tests/integration/libssh.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ pkcs11_provider_setup()
 
 libssh_setup()
 {
-    title PRAM "Clone, setup and build libssh"
+    title PARA "Clone, setup and build libssh"
 
     git clone https://gitlab.com/libssh/libssh-mirror.git \
```
Comment (The-Mule) [inline tests/integration/libssh.sh:52]: Can you also fix the link please?

Comment (The-Mule) [reply tests/integration/libssh.sh:52]: https://git.libssh.org/projects/libssh.git/

Comment (Jakuje) [reply tests/integration/libssh.sh:52]: This git forge is running on some Andreas servers. I do not know how much traffic the cloning from CI is going to generate so I would prefer to keep the gitlab mirror as that is automatically updated every hour or so and is more likely to be present.

Comment (The-Mule) [reply tests/integration/libssh.sh:52]: I see, yes, that makes sense, please disregard my comment :). It looks like this mirror is up to date.

**Review by simo5 — APPROVED** [review]
LGTM

**Comment by simo5** [comment]
With recent rebuild of gnutls in rawhide now all tests pass, so this is finally fixed.
Merging

**Pull request merged**